core/src/main/resources/hudson/model/Messages.properties

```diff
@@ -221,6 +221,7 @@ Run.BuildAborted=Build was aborted
 Run.MarkedExplicitly=This record is explicitly marked to be kept.
 Run.Permissions.Title=Run
 Run.running_as_=Running as {0}
+Run.running_as_SYSTEM=This build is running using the Jenkins-internal SYSTEM authentication with no permission restrictions.
```

Comment:
I can only +1 the idea to link a page / blog post about the recommended way to configure the AuthQueue.

Reply:
Right, but probably not in every build log -- technically adding a `HyperlinkNote` is trivial, but it seems very noisy.